components/ui/text-input/select/SelectInput.tsx

```diff
@@ -76,17 +77,26 @@ const SelectInput: React.FC<SelectInputComponentProps<any>> = ({
     onPress();
   };
 
+  const showLoader = data.length === 0;
```

Comment:
don't think it is good condition. maybe make data undefined when data is not there? We got case in payments where data is empty (list of recipient i group has only on person). It shows like it tries to fetch the list but the list is already fetched but it's empty.